### ISSUE TITLE
Upgrade to nuxt/kit 3.4

### ISSUE THREAD

**Issue by localusercamp** [nuxt-modules/tailwindcss#654]
# Problem
TailwindCSS nuxt module uses nuxt/kit 3.3 which requires consola below v3. This leads to dependency mismatches in projects with newly released nuxt 3.4 version.

# How to solve
This module needs greater version of nuxt/kit (3.4) to resolve this [issue](https://github.com/nuxt/nuxt/issues/20209).

**Comment by ineshbose** [comment]
I agree - quite important! Maintainers are a bit busy atm, but PR is most welcome 🙂
